### PR TITLE
document how to imperatively add and remove listeners

### PR DESCRIPTION
**Pull request by kaycebasques**
@arthurevans just a little thing I encountered while perusing [SO](http://stackoverflow.com/questions/34505363/polymer-1-x-imperatively-adding-event-listener/34530249#34530249)...

### CODE REVIEW

File: 1.0/docs/devguide/events.md [hunk 1]
```diff
@@ -102,6 +102,17 @@ So the attribute `on-core-signal-newData` sets up a listener for `core-signal-ne
...
+
+Use [automatic node finding](local-dom.html#node-finding) and the 
+convenience methods 
+[`listen`](/{% polymer_version_dir %}/api/#Polymer.Base:method-listen) and 
+[`unlisten`](/{% polymer_version_dir %}/api/#Polymer.Base:method-unlisten).
+
+    this.listen(this.$.myButton, 'tap', 'onTap');
+
+    this.unlisten(this.$.myButton, 'tap', 'onTap');
+
```
Comment (arthurevans) [inline 1.0/docs/devguide/events.md]: LG. Might want to add: the listener callback is invoked with `this` set to the element instance. 

If you add a listener imperatively, you need to remove it imperatively. This is commonly done in the `attached` and `detached` callbacks. If you use the `listeners` object or annotated event listeners, {{site.project_title}} automatically adds and removes the event listeners.

Comment (kaycebasques) [inline 1.0/docs/devguide/events.md]: :heavy_check_mark:

**Comment by kaycebasques** [comment]
OK @arthurevans, added your feedback

**Comment by arthurevans** [comment]
LGTM, thanks.

**Pull request merged**